homeassistant/components/fitbit/const.py

```diff
@@ -276,35 +290,35 @@ class FitbitSensorEntityDescription(SensorEntityDescription, FitbitRequiredKeysM
 
 FITBIT_MEASUREMENTS: Final[dict[str, dict[str, str]]] = {
     "en_US": {
-        ATTR_DURATION: TIME_MILLISECONDS,
-        ATTR_DISTANCE: "mi",
-        ATTR_ELEVATION: LENGTH_FEET,
-        ATTR_HEIGHT: "in",
-        ATTR_WEIGHT: "lbs",
-        ATTR_BODY: "in",
-        ATTR_LIQUIDS: "fl. oz.",
-        ATTR_BLOOD_GLUCOSE: f"{MASS_MILLIGRAMS}/dL",
+        ATTR_DURATION: UnitOfTime.MILLISECONDS,
+        ATTR_DISTANCE: UnitOfLength.MILES,
+        ATTR_ELEVATION: UnitOfLength.FEET,
+        ATTR_HEIGHT: UnitOfLength.INCHES,
+        ATTR_WEIGHT: UnitOfMass.POUNDS,
+        ATTR_BODY: UnitOfLength.INCHES,
+        ATTR_LIQUIDS: UnitOfVolume.FLUID_OUNCES,
+        ATTR_BLOOD_GLUCOSE: f"{UnitOfMass.MILLIGRAMS}/dL",
         ATTR_BATTERY: "",
     },
     "en_GB": {
-        ATTR_DURATION: TIME_MILLISECONDS,
-        ATTR_DISTANCE: "kilometers",
-        ATTR_ELEVATION: "meters",
-        ATTR_HEIGHT: "centimeters",
-        ATTR_WEIGHT: "stone",
-        ATTR_BODY: "centimeters",
-        ATTR_LIQUIDS: "milliliters",
+        ATTR_DURATION: UnitOfTime.MILLISECONDS,
+        ATTR_DISTANCE: UnitOfLength.KILOMETERS,
+        ATTR_ELEVATION: UnitOfLength.METERS,
+        ATTR_HEIGHT: UnitOfLength.CENTIMETERS,
+        ATTR_WEIGHT: UnitOfMass.STONES,
+        ATTR_BODY: UnitOfLength.CENTIMETERS,
+        ATTR_LIQUIDS: UnitOfVolume.MILLILITERS,
```

Comment:
I realise you haven't changed this, but I question these GB units. If weight is in stones, height should be in feet. Distance would most likely be miles.

Reply:
The units need to match what comes back from fitbit api. So I wouldn't change them.